.devcontainer/scripts/postCreateCommand.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
Consider renaming the container scripts folder to be source-build-scripts to provide more context.